metadata.json

```diff
@@ -18,7 +18,7 @@
     { "operatingsystem": "FreeBSD" }
   ],
   "dependencies": [
-    {"name":"puppetlabs/concat","version_requirement":">= 1.0.1 <3.0.0"},
+    {"name":"puppetlabs/concat","version_requirement":">= 1.0.1 <=4.0.1"},
```

Comment:
Is there any reason to not push the concat version req range out to `< 5.0.0`?

Reply:
Yes and no, I can only say I have tested with 4.0.1
That being said, minors shouldn't introduce breaking changes....